lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2004/data/ashrae_90_1_2004.spc_typ.json

```diff
@@ -8012,11 +8012,11 @@
       "occupancy_schedule": "HotelLarge BLDG_OCC_SCH",
       "occupancy_activity_schedule": "HotelLarge ACTIVITY_SCH",
       "is_residential": null,
-      "ventilation_standard": "ASHRAE 62.1-2007",
-      "ventilation_primary_space_type": "Retail",
-      "ventilation_secondary_space_type": "General Sales",
-      "ventilation_per_area": 0.12,
-      "ventilation_per_person": 7.5,
+      "ventilation_standard": "ASHRAE 62.1-1999",
+      "ventilation_primary_space_type": "Retail Stores, Sales Floors, and Show Room Floors",
+      "ventilation_secondary_space_type": "Basement and street",
+      "ventilation_per_area": 0.3,
+      "ventilation_per_person": 0.0,
```

Comment:
The ventilation primary and secondary space types were changed based on 62.1 and the corresponding ventilation rates were updated.